node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const file = process.argv;
+
+const cat = () => {
+  let all = '';
+  for (let i = 2; i < file.length; i++) {
+    fs.readFile(file[i], 'utf-8', (err, data) => {
+      if (err) throw err;
+      all += data;
+    });
+  }
+  console.log(all);
+};
+
+cat();
```

Comment:
Good attempt! But this won't print out all of the user inputted files in the correct order every time.